### DISCUSSION TITLE
keyball61の片手のみ入力が出来ません。

### DISCUSSION THREAD

**Discussion by expgolemclone** [Yowkees/keyball#134] [Q&A]
この度は、keyball61(ProMicro用ピンヘッダ＋ピンソケット)を購入させていただきました。
しかし、片方のボールがない側は正常に動作するのですが、ボールのある右側のみすべてのキーが反応しません。
どのようにすればよいかご指南して頂けましたら幸いです。

・試したこと
1. ビルドガイドに沿って"10章の正規ファームウェア書き込み"まで作業をしました。
2. ダイオードの極性を確認しました。
3. 遊舎工房の記事の①から➂までは確認しました。
https://yushakobo.zendesk.com/hc/ja/articles/360043073834-%E3%82%AD%E3%83%BC%E5%85%A5%E5%8A%9B%E3%81%8C%E5%85%A8%E7%84%B6%E3%81%A7%E3%81%8D%E3%81%AA%E3%81%84

・動作しない原因の心当たり
1. 12Pピンソケットが破損してしまったので、代用として12ピンヘッダを使ってpromicroを固定した。
![IMG_0641](https://user-images.githubusercontent.com/108988326/190573938-20e57393-4611-4edc-9136-e30087da64e7.JPG)

以上となります。お手数おかけしますが宜しくお願いします:sob:

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

すべてのキーが反応せずOLEDも表示されていないようですので、ProMicro周辺から探ってみましょう。

いただいたお写真のはんだ付け部と下のピンがすべて導通しているか確認できますでしょうか？

今後何度かやりとりが出てきそうですので、白銀ラボホームページのお問い合わせからご連絡いただければ返信スムーズと思いますのでご連絡いただければ幸いです🙏
